config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
+  resources :languages do
+    delete 'remove'
+  end
   root 'home#index'
```

Comment:
The `root` route should always go at the top of the `routes` file since Rails looks for matching routes in order and stops when it hits a matching path.